testem.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  disable_watching: true,
+  framework: 'mocha',
+  launch_in_ci: [
+    'Firefox'
+  ],
+  launch_in_dev: [
+    'Chrome'
+  ],
+  test_page: 'tests/index.html?hidepassed&coverage'
+}
```

Comment:
It is linted as well as provides the ability to add comments if we wish. Also new Ember addons ship with the `..js` not a `.json` so this is more in-line with the Ember way.